applicationcharge.go

```diff
@@ -39,41 +39,41 @@ type ApplicationCharge struct {
 }
 
 // ApplicationChargeResource represents the result from the
-// admin/application_charges{/X{/activate.json}.json}.json endpoints.
+// admin/api/2019-04/application_charges{/X{/activate.json}.json}.json endpoints.
```

Comment:
Was this change intentional? Same with the one on line 48?

Reply:
I had made it while passing over everything and trying to address the overall version issue, not really sure if it's more or less clear in a comment like this over time...  I've reverted both lines that you've brought up here.